### ISSUE TITLE
FLOPS in ResNet50_ImageNet

### ISSUE THREAD

**Issue by miltonmondal** [sovrasov/flops-counter.pytorch#94]
Getting 4.12B flops using your code where all research papers mentioned 4.09B flops for this configuration

 (pytorch default 76.15% test accuracy for pretrained model)

**Comment by jkhu29** [comment]
[fvcore](https://github.com/facebookresearch/fvcore) will get the output of 4.09G, but it will also print

```shell
Skipped operation aten::batch_norm 53 time(s)
Skipped operation aten::max_pool2d 1 time(s)
Skipped operation aten::add_ 16 time(s)
Skipped operation aten::adaptive_avg_pool2d 1 time(s)
```

Perhaps those papers ignore the computation of some of the operators.

**Comment by sovrasov** [comment]
@jkhu29 you're right! ptflops also considers batch norms and poolings as non-zero ops, that's why it outputs slightly greater numbers than expected.

**Comment by oyryrookie** [comment]
> [@jkhu29](https://github.com/jkhu29) you're right! ptflops also considers batch norms and poolings as non-zero ops, that's why it outputs slightly greater numbers than expected.

How to ignore the batch norms and poolings ops in ptflops?

**Comment by sovrasov** [comment]
> > [@jkhu29](https://github.com/jkhu29) you're right! ptflops also considers batch norms and poolings as non-zero ops, that's why it outputs slightly greater numbers than expected.
> 
> How to ignore the batch norms and poolings ops in ptflops?

You can pass all the nn.Modules to exclude via `ignore_modules` parameter, see https://github.com/sovrasov/flops-counter.pytorch?tab=readme-ov-file#usage-tips-1
